src/plugins/i18n.ts

```diff
@@ -26,7 +26,8 @@ const startingLocale = getStartingLocale()
 const i18n: VueI18n = new VueI18n({
   locale: startingLocale,
   fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || 'en',
-  messages: {}
+  messages: {},
+  silentFallbackWarn: true
```

Comment:
Not sure this belong in the this PR

Reply:
The warn is written only in development in the console and was flooding the console so i disabled the warning of fallback of key this don't disable missing key warning

Reply:
Both warnings are important in dev. Ultimately there should be none, in any case this doesn't belong in this PR as it is a "design" choice.